docs/automated-tests/test-execution/quick-execution.md

```diff
@@ -1,55 +1,57 @@
 ---
-title: How to Execute Test
+title: How to Execute Tests
 page_title: Execute Your First Test in Test Studio
-description: "Test Studio Quick test Execution. Run a Test Studio test. Choose browser to execute the test against."
+description: "Learn how to execute quick tests in Test Studio, modify the quick tests, and access the test results"
 position: 0
 ---
-# How to Execute Test
+# How to Execute Tests
 
-__Quick Test Execution__ in Test Studio is a key feature, which helps in fine tuning the recorded actions, adjust test and step settings, to eventually introduce stable and consistent behavior of the test runs in test list mode, executed on different environments.
+__Quick Test Execution__ helps you fine tune the recorded actions, test, and step settings. It helps you achieve stable and consistent test behavior in test list mode when you execute the test in different environments.
 
-This article will guide you through the execution process in details.
```

Comment:
Try to use Present Simple as much as possible, see  https://testdocs.telerik.com/style-guide/basic-rules-and-guidelines#use-simple-tenses